tests/bud.bats

```diff
@@ -608,7 +608,8 @@ load helpers
   run_buildah bud --signature-policy ${TESTSDIR}/policy.json -t ${target} ${TESTSDIR}/bud/volume-ownership
   run_buildah from --quiet --signature-policy ${TESTSDIR}/policy.json ${target}
   cid=$output
-  run_buildah run $cid test-ownership
+  run_buildah run $cid stat -c "%U %G" /vol/subvol
```

Comment:
So, I'm a complete ignoramus about the VOLUME directive, and I could test this (and will, when I get some time this morning), but: how would this behave differently if the Containerfile _did not_ specify `VOLUME`? Is there any way to check that `/vol/subvol` is actually being treated as a volume, and not just a regular subdirectory in the image?

Reply:
tldr; I wasn't able to find a good way.

Volumes are mount points in containers so I figured I could use `mountpoint` to test the directory, but ...
```sh
/ # mount
...
/dev/mapper/luks-5825177c-f5c0-47c1-9865-db72a9bd7528 on /vol/subvol type btrfs (rw,seclabel,nosuid,nodev,relatime,ssd,space_cache,subvolid=257,subvol=/home/ncarboni/.local/share/containers/storage/volumes/b2efdcb7017c80a10b244942da38e4620d3ec5af0de90a9fee020ea44118679c/_data)
...
/ # mountpoint /vol/subvol/
/vol/subvol/ is not a mountpoint
```

:worried: 

I can't seem to get consistent results from `stat -c "%i` or `stat -f` inside the container either. We could parse `mount` or `df`, but ... gross.

Reply:
Understood. Thanks for having looked into it.

Reply:
@edsantiago The VOLUME directive tells the container engines to allocate a UNNAMED volume on disk and mount it at the /vol/submount directory.  

The problem was podman was creating the new volume with the wrong owner/permissions and then mounting over the /vol/submount.  The directory went from ownership testuser testgroup to root root.

Reply:
@carbonin, I can confirm: `mountpoint -n`, `-d`, `blkid`, `stat -c %i`, none of those (or others I've forgotten) produce any useful distinction.